app/models/katello/product.rb

```diff
@@ -199,6 +199,10 @@ def product_content_by_id(content_id)
       product_contents.joins(:content).where("#{Katello::Content.table_name}.cp_content_id = ?", content_id).first
     end
 
+    def product_content_by_label(content_label)
```

Comment:
wonder if this code is better placed in importable_repositories

Reply:
I will move it

Reply:
Fixed.